tests/tasks/assert_certificate_parameters.yml

```diff
@@ -10,6 +10,16 @@
       - result.stat.exists
     fail_msg: "Certificate file '{{ cert['path'] }}' does not exist."
 
+- name: Verify certificate file owner and group
+  assert:
+    that:
+      - result.stat.pw_name == (cert['owner'] | default('root'))
```

Comment:
`User name (or user id) for the certificate and key files` - this test will fail if `owner` was given as a user id.  You should add a comment in the test to warn future test writers.